frontend/src/metabase/components/SandboxedPortal.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+function stop(event) {
+  event.stopPropagation();
+}
+
+function SandboxedPortal({ children, container }) {
```

Comment:
React portals bubble events _through_ the portal, which breaks our wrapper `Triggerable` component. I found this solution here: https://github.com/facebook/react/issues/11387#issuecomment-355395803

Reply:
I like the fix! Maybe leave this as a comment in the code? The code is pretty self-explanatory, but I think "Sandboxed" is an overloaded term so there might be some confusion.